app/models/github_repository.rb

```diff
@@ -32,6 +32,12 @@ def get_starter_code_from(source)
 
   # Public
   #
+  def self.present?(client, full_repo_name)
```

Comment:
Can we update this to just be `full_name`? That way it matches the API response we get.